src/sequenceParser/ItemStat.hpp

```diff
@@ -45,7 +45,7 @@ class ItemStat
 	long long sizeOnDisk; /// size on hard-drive (takes hardlinks into account)
 	long long accessTime;
 	long long modificationTime;
-	long long creationTime;
+	long long lastChangeTime;
```

Comment:
`lastChangeTime = statInfos.st_ctime;`
http://linux.die.net/man/2/stat
`time of last status change`

Maybe need a more explicit variable name and also some doc.